app/views/katello/api/v2/products/base.json.rabl

```diff
@@ -12,6 +12,8 @@ attributes :sync_summary
 attributes :gpg_key_id
 attributes :redhat? => :redhat
 
+attributes :available_content => :available_content, :if => params[:include_available_products]
```

Comment:
Added to [API docs](https://github.com/Katello/katello/pull/5242/files#diff-cedcc866cbdbe325575a0c9eee8a1fe3R29)